<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 43, 43);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 700 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#ffffff;&quot;&gt;Tic Tac Toe Game&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAI">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 12pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
    </property>
    <property name="text">
     <string>AI Opponant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMulti">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 12pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
    </property>
    <property name="text">
     <string>Multiplayer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfile">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 12pt &quot;Segoe UI&quot;;
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signout">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3498db; /* Background color */
    color: #ffffff;            /* Text color */
    border: 2px solid #ffffff; /* Border color and width */
    border-radius: 10px;       /* Edges sharpness */
    padding: 5px;              /* Padding for better spacing */

}</string>
    </property>
    <property name="text">
     <string>Sign out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
